website/content/docs/services/discovery/dns-configuration.mdx

```diff
@@ -33,6 +33,9 @@ You can specify a list of addresses in the agent's [`recursors`](/consul/docs/ag
 
 Nodes that query records outside the `consul.` domain resolve to an upstream DNS. You can specify IP addresses or use `go-sockaddr` templates. Consul resolves IP addresses in the specified order and ignores duplicates.
 
+> [!WARNING]  
+> We recommend configuring your DNS resolvers to point the `consul.` domain towards your Consul DNS server(s). Misconfigurations can result in queries for the `consul.` domain being routed outside of your network instead, leaking DNS queries to root DNS servers.
```

Comment:
Since the rest of the documentation refers it as `consul.` I thought it would be easier for readers to follow the same path however I'm open to change it if needed.

Reply:
Good point, added a specific action link for reference: https://github.com/hashicorp/consul/pull/21250/commits/d27b6b6711316e5643198f20c4e49b0b49371c0d